packages/app/components/Queue/Student/StudentQueueList.tsx

```diff
@@ -115,7 +117,7 @@ export default function StudentQueueList({
         queue: newQuestionsInQueue,
       });
     },
-    [studentQuestion?.id, questions, mutateQuestions]
+    [studentQuestionStatus, studentQuestionId, questions, mutateQuestions]
```

Comment:
this fixes the issue with editing while in priority queue

Reply:
got it